### ISSUE TITLE
PDF built in undefined

### ISSUE THREAD

**Issue by pepelsbey** [shower/cli#23]
```bash
$ shower pdf
  ✔ Creating PDF in progress
PDF built in undefined 🎉 [in 2s]
$ shower --version
0.2.4
```

**Comment by avdeev** [comment]
double https://github.com/shower/shower/issues/334?

**Comment by pepelsbey** [comment]
Regression, I guess.

**Comment by nikolay-govorov** [comment]
Thank you, took

**Comment by nikolay-govorov** [comment]
Fixed in version `0.2.5`
